### PR TITLE
Feat/linkedin properties

### PR DESCRIPTION
**Pull request by RobinStut**

### CODE REVIEW

File: src/heads-up/components/app-tooltip.vue [hunk 1]
```diff
@@ -17,22 +17,22 @@
 </template>
 
 <script>
-  import getTheme from '../lib/theme'
+import getTheme from "../lib/theme";
 export default {
```
Comment (Maikxx) [inline src/heads-up/components/app-tooltip.vue:22]: enter here

File: src/heads-up/components/property-data.vue [hunk 2]
```diff
@@ -54,8 +54,8 @@ export default {
       }
 
       if (this.type === "og:image") {
-        result += this.isCorrectUrl(this.type, this.exist)
-        result += this.isBigImg(this.type, this.hasVariation, this.size, this.requiredSizes)
+          result += this.value === null ? this.isCorrectUrl(this.type, this.exist) : ''
```
Comment (Maikxx) [inline src/heads-up/components/property-data.vue]: `result += Boolean(this.value) ? '' : this.isCorrectUrl(this.type, this.exist)`

File: src/heads-up/components/property-data.vue [hunk 3]
```diff
@@ -76,11 +76,11 @@ export default {
       return `The content has more than ${length} characters. Consider shorten your content. `
     },
     isRequired(tag) {
-      return `The ${tag} is required to create an unfurling link on the platform. `
+      return `The ${tag} is required to create an unfurling link on this platform. `
     },
     isBigImg(tag, smallVariant, imgSize, requiredSize) {
-      if (smallVariant) {
-        return`The ${tag} is ${imgSize.width} by ${imgSize.height}px, where ${requiredSize.width} by ${requiredSize.height}px is required for a big unfurling preview. `
+      if (smallVariant && requiredSize.variation.width > imgSize.width && requiredSize.variation.height > imgSize.height) {
```
Comment (Maikxx) [inline src/heads-up/components/property-data.vue]: Make this a variable with a nice name to make it easier to read this

File: src/heads-up/views/linkedin.vue [hunk 4]
```diff
@@ -47,33 +106,91 @@
...
+import WarningIcon from "../assets/icons/warning.svg";
 import { mapState } from "vuex";
-import { ExternalLink, PanelSection, PropertiesList } from "../components";
+import {
+  ExternalLink,
+  PanelSection,
+  PropertiesList,
+  AppTooltip,
+  PropertyData
+} from "../components";
```
Comment (Maikxx) [inline src/heads-up/views/linkedin.vue:118]: enter here

File: src/heads-up/views/linkedin.vue [hunk 5]
```diff
@@ -130,4 +272,7 @@ export default {
 .linkedin__preview-caption {
```
Comment (Maikxx) [inline src/heads-up/views/linkedin.vue:273]: This can maybe be gone?

File: src/heads-up/views/linkedin.vue [hunk 6]
```diff
@@ -130,4 +272,7 @@ export default {
...
   color: var(--label-color);
 }
+.properties-item__icon {
```
Comment (Maikxx) [inline src/heads-up/views/linkedin.vue]: .linkedin .properties-item__icon

File: src/heads-up/components/property-data.vue (hunk 3 above)
Comment (Maikxx) [inline src/heads-up/components/property-data.vue]: dit kan in een const

File: src/heads-up/components/property-data.vue (hunk 2 above)
Comment (Maikxx) [inline src/heads-up/components/property-data.vue]: boolean check here

File: src/heads-up/components/app-tooltip.vue [hunk 7]
```diff
@@ -17,22 +17,22 @@
 </template>
 
 <script>
-  import getTheme from '../lib/theme'
+import getTheme from "../lib/theme";
```
Comment (Maikxx) [inline src/heads-up/components/app-tooltip.vue:20]: enter afterhere

**Review by Maikxx — CHANGES_REQUESTED** [review]

File: src/heads-up/views/linkedin.vue (hunk 5 above)
Comment (RobinStut) [reply src/heads-up/views/linkedin.vue:273]: Wordt wel gebruikt op regel 18

**Review by Maikxx — APPROVED** [review]

**Pull request merged**